test/unit/foreman/renderer/snapshots/ProvisioningTemplate/script/Windows_peSetup.cmd.windows10_dhcp.snap.txt

```diff
@@ -0,0 +1,97 @@
+
+@setlocal enableextensions enabledelayedexpansion
+@echo off
+set WGET=wget64.exe
+
+zerombr
```

Comment:
Looks like it's using a Red Hat partitioning table. That's probably a bug in the factory.

Reply:
I have added a partition table for windows. I am absolutely not sure if I added a correct format, though.